stream/chunk_writer.js

```diff
@@ -21,10 +21,19 @@
 'use strict';
 
 var inherits = require('util').inherits;
+var inspect = require('util').inspect;
 var Transform = require('stream').Transform;
+var WrappedError = require('error/wrapped');
 
 var toBufferTuple = require('../interface').toBufferTuple;
 
+var ChunkWriteError = WrappedError({
+    type: 'chunk-write',
```

Comment:
no other typed errors in bufrw are namespaced... open to converting before 1.0 if there's benefit tho, not in this PR